misc/make-deps.sh

```diff
@@ -104,6 +106,23 @@ if [ $travis -eq 0 ]; then
 	fi
 fi
 
+if [ $travis -eq 1 ]; then
```

Comment:
Really? Can't it be installed from some package repo?

Reply:
Apparently not :(